Description: Adjust Fortran MOD installation path in pkg-config file.
Author: Andrius Merkys <merkys@debian.org>
Forwarded: not-needed
--- a/meson.build
+++ b/meson.build
@@ -69,7 +69,7 @@
     install_dir: get_option('datadir')/'licenses'/meson.project_name()
   )
 
-  module_id = meson.project_name() / fc_id + '-' + fc.version()
+  module_id = meson.project_name()
   meson.add_install_script(
     find_program(files('config'/'install-mod.py')),
     get_option('includedir') / module_id,
